docker/README.md

```diff
@@ -14,7 +14,7 @@
 
     ## init mysql table before the first running
     cd /tmp && \
-    git clone https://github.com/open-falcon/falcon-plus && \
+    git clone https://github.com/open-falcon/falcon-plus --depth=1 && \
```

Comment:
```suggestion
    git clone --depth=1 https://github.com/open-falcon/falcon-plus && \
```

According to `git-clone(1)`, `--depth` should be given before the `<repository>`.

```
SYNOPSIS
       git clone [--template=<template_directory>]
                 [-l] [-s] [--no-hardlinks] [-q] [-n] [--bare] [--mirror]
                 [-o <name>] [-b <name>] [-u <upload-pack>] [--reference <repository>]
                 [--dissociate] [--separate-git-dir <git dir>]
                 [--depth <depth>] [--[no-]single-branch] [--no-tags]
                 [--recurse-submodules[=<pathspec>]] [--[no-]shallow-submodules]
                 [--jobs <n>] [--] <repository> [<directory>]
```